### PR TITLE
CONSOLE-3642: Add edit labels action and Cypress test to CronTab list page

### PR DESCRIPTION
**Pull request by TheRealJon**
- Add "Edit Labels" action to CronTab list page kebab using dynamic plugin SDK hook
- Add test to ensure label editing works as expected
- Fix i18n issue where keys were not being generated for items that were not string literals

### CODE REVIEW

File: integration-tests/e2e/list-page.cy.ts [hunk 1]
```diff
@@ -40,6 +40,28 @@ describe(`${PLUGIN_NAME} list page test`, () => {
...
+    cy.log("Updating the labels of the new CronTab from the list page");
+    cy.visit(namespacedListPageURL);
+    listPage.rows.shouldBeLoaded();
+    listPage.rows.shouldExist("my-new-cron-object");
+    listPage.rows.clickKebabAction("my-new-cron-object", "Edit Labels");
+    modal.shouldBeOpened();
+    cy.byLegacyTestID("modal-title").should("have.text", "Edit labels");
+    modal.submitShouldBeEnabled();
+    cy.byTestID("tags-input").type("key1=value1{enter}");
+    cy.get(".tag-item-content .pf-c-label__content").should(
```
Comment (jhadvig) [inline integration-tests/e2e/list-page.cy.ts:52]: I would avoid accessing elements like this. Lets ratherre/use helpers that we already have in console repo. I've reused them in https://github.com/openshift/crontab-plugin/pull/8/files#diff-b1dce0141c3765e3e888abdc79f28116a9e74498207b3d6e1aa74233c794686b

File: integration-tests/e2e/list-page.cy.ts [hunk 2]
```diff
@@ -30,7 +30,7 @@ describe(`${PLUGIN_NAME} list page test`, () => {
     listPage.rows.countShouldBe(0);
   });
 
-  it("Create and delete a new CronTab from the list page", () => {
+  it("Create, update labels, and delete a CronTab from the list page", () => {
```
Comment (jhadvig) [inline integration-tests/e2e/list-page.cy.ts:33]: nit
```suggestion
  it("Create, update labels and delete a CronTab from the list page", () => {
```

File: integration-tests/views/labels-modal.ts [hunk 3]
```diff
@@ -0,0 +1,9 @@
+export const labelsModal = {
+  getTitle: () => cy.byLegacyTestID("modal-title"),
```
Comment (jhadvig) [inline integration-tests/views/labels-modal.ts:2]: this helper is not used anywhere

File: integration-tests/e2e/list-page.cy.ts [hunk 4]
```diff
@@ -71,19 +83,43 @@ describe(`${PLUGIN_NAME} list page test`, () => {
...
+    cy.log("Edit labels");
+    cy.visit(namespacedListPageURL);
+    listPage.rows.shouldBeLoaded();
+    listPage.rows.shouldExist("my-new-cron-object");
+    listPage.rows.clickKebabAction(
+      "my-new-cron-object",
+      KEBAB_ACTION_EDIT_LABELS_ID
+    );
+    modal.shouldBeOpened();
+    cy.byLegacyTestID("modal-title").should("have.text", "Edit labels");
```
Comment (jhadvig) [inline integration-tests/e2e/list-page.cy.ts:106]: for this we already have a helper, check `modalTitleShouldContain()`

**Review by jhadvig — COMMENTED** [review]
Changes look good 👍 
Adding a few comments

**Comment by TheRealJon** [comment]
@jhadvig I added some changes to the e2e tests

**Review by jhadvig — APPROVED** [review]
/lgtm
/approve

**Comment by jhadvig** [comment]
QE Approver:
/assign @yanpzhan 
Docs Approver: 
/assign @opayne1 
PX Approver:
/assign @RickJWagner

**Comment by TheRealJon** [comment]
@jhadvig CI wants the target version to be 4.15.0, but that is not an available option for the Dynamic Plugin component in Jira.

**Pull request merged**